net-misc/stargazer/stargazer-2.408-r1.ebuild

```diff
@@ -0,0 +1,535 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PROJECTS="sgconv rlm_stg rscriptd sgauth sgconf sgconf_xml stargazer"
+
+STG_MODULES_AUTH="always_online internet_access freeradius"
+STG_MODULES_CAPTURE="ipq ether netflow"
+STG_MODULES_CONFIG="sgconfig rpcconfig"
+STG_MODULES_OTHER="ping smux remote_script"
+STG_MODULES_STORE="files firebird mysql postgres"
+
+declare -A MODULES
+MODULES=( [module_auth_always_online]="authorization\/ao:mod_ao"
+	[module_auth_internet_access]="authorization\/inetaccess:mod_ia"
+	[module_auth_freeradius]="other\/radius:mod_radius"
+	[module_capture_ipq]="capture\/ipq_linux:mod_cap_ipq"
+	[module_capture_ether]="capture\/ether_linux:mod_cap_ether"
+	[module_capture_netflow]="capture\/cap_nf:mod_cap_nf"
+	[module_config_sgconfig]="configuration\/sgconfig:mod_sg"
+	[module_config_rpcconfig]="configuration\/rpcconfig:mod_rpc"
+	[module_other_ping]="other\/ping:mod_ping"
+	[module_other_smux]="other\/smux:mod_smux"
+	[module_other_remote_script]="other\/rscript:mod_remote_script"
+	[module_store_files]="store\/files:store_files"
+	[module_store_firebird]="store\/firebird:store_firebird"
+	[module_store_mysql]="store\/mysql:store_mysql"
+	[module_store_postgres]="store\/postgresql:store_postgresql"
+)
+
+declare -A INIT
+INIT=(	[module_store_files]="11d"
+	[module_store_firebird]="11d;s/need net/need net firebird/"
+	[module_store_mysql]="11d;s/need net/need net mysql/"
+	[module_store_postgres]="11d;s/need net/need net postgresql/"
+)
+
+MY_P="stg-${PV}"
+
+inherit eutils linux-info user
+
+DESCRIPTION="Billing system for small home and office networks"
+HOMEPAGE="http://stg.dp.ua/"
+SRC_URI="http://stg.dp.ua/download/server/${PV}/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+REQUIRED_USE="stargazer? ( ^^ ( module_store_files module_store_firebird module_store_mysql module_store_postgres ) )"
+
+RDEPEND="module_config_rpcconfig? (
```

Comment:
put `module_config_rpcconfig` on the next line for better consistency

Reply:
Done.